javascript/javascript1/week1/ez_namey.js

```diff
@@ -0,0 +1,10 @@
+const firstWords = ["Easy", "Awesome", "Corporate", "Severe", "Smiling", 
+    "Heavenly", "Childlike", "Shy", "Anxious", "Ugliest", "Accurate"];
```

Comment:
Minor inconsequential 'bug' the array has 11 items, ranging from index 0-10. The highest number generated at random is 9, therefore index 10(Accurate) will never he hit.

As a fun small side quest, you could consider making the random number generator depend on the length of your array. Thereby ensuring that all items get a chance to be featured, and make it robust if in the future you want to change the number of items in the array ;)

Let me know if you need any help with this.

Reply:
Wow! Nice catch! Fixed it in next commit